src/components/StarRating/index.tsx

```diff
@@ -8,44 +10,46 @@ const cx = classNames.bind(styles);
 const { star } = SVGS;
 
 type StarRatingProps = {
-  size: 'small' | 'medium' | 'large';
+  size?: 'small' | 'medium' | 'large';
   rating?: number;
   readonly?: boolean;
   onChange?: (rating: number) => void;
 };
 
-const StarRating = ({ size, onChange, rating = 0, readonly = false }: StarRatingProps) => {
+const StarRating = ({ onChange, rating = 0, size = 'medium', readonly = false }: StarRatingProps) => {
   const MAX_RATING = 5;
   const OFFSET = 1;
 
+  const [filledStarId, setFilledStarId] = useState(rating);
+
   const handleStarClick = (starId: number) => {
     if (!onChange || readonly) return;
     const newRating = starId + OFFSET;
-    onChange(newRating);
+
+    setFilledStarId(newRating);
+    onChange && onChange(newRating);
```

Comment:
```suggestion
    onChange(newRating);
```
앞에서 onChange가 없을때 이미 리턴을 시켜줘서, 이부분은 없어도 될거같습니다!

Reply:
holy...